CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 [//]: # (current developments)
 
+## 22.11.1 (2022-12-05)
+
+### Bug fixes
+
+* Restore default virtual package specs as in 22.9.0 (#12148)
+  - re-add `__unix`/`__win` packages
+  - restore `__archspec` version/build string composition
+
+### Other
+
+* Skip test suite for non-code changes. (#12141)
+
+
+
```

Comment:
Contributors are missing? Rerun with authors and changelog actions simultaneously?

Reply:
Odd, retrying..

Reply:
That did the trick.